resources/jenkins-repackage-resources.sh

```diff
@@ -24,15 +24,13 @@ mkdir -p ${TEMPORARY_DIRECTORY}
 resources/repackage-easyrsa.sh
 
 export OS=$(get_os)
-export ARCHITECTURES=${ARCHITECTURES:-"amd64"}  #"amd64 arm arm64 ppc64le"
+export ARCHITECTURES=${ARCHITECTURES:-"amd64 arm64"}  #"amd64 arm arm64 ppc64le"
```

Comment:
Why no s390x here? Is upstream not shipping s390x packages?

Reply:
Indeed there is no s390x for etcd: https://github.com/coreos/etcd/releases needed for flannel here: https://github.com/juju-solutions/kubernetes-jenkins/blob/master/resources/repackage-flannel.sh#L32